sentry-kotlin-extensions/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+import net.ltgt.gradle.errorprone.errorprone
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
```

Comment:
you need to apply detekt plugin

https://github.com/getsentry/sentry-java/blob/main/sentry-android-timber/build.gradle.kts#L93-L96